src/app/main/connection-window/connection-window.component.html

```diff
@@ -58,18 +72,18 @@
             </div>
           </div>
         </div>
-        <button class="btn btn-primary connect-button" (click)="onConnect()">Connect</button>
+        <button #myDiv class="btn btn-primary connect-button" (click)="onConnect()">Connect</button>
```

Comment:
` #myDiv` ?

Reply:
This is used so we can do @ViewChild to get the HTMLElement.